### ISSUE TITLE
Corrupted .mov from iOS to Android

### ISSUE THREAD

**Issue by aiya000** [localsend/localsend#1958]
### Describe the bug

We succeed fetching movies from iOS18 (iPhone15) to my Android15 (Google Pixel Pro 9 Fold), but fetched movies are corrupted.

If I try to view corrupted movies, depending on media player app, failed to view, or succeed.
Also on my photo app (FStop), generating the thumbnails is failed.

Please fix this issue.
Thanks!

### To reproduce

1. On iOS18, send some movie
2. On Android15, fetch the movie
3. On Android15, check the movie on a photo app, e.g. FStop
4. The thumbnail is grey outed

### Expected behavior

Getting not corrupted movies.

### Desktop (please complete the following information)

Not checked.

### Smartphone (please complete the following information)

- Device: iPhone15, Pixel 9 Pro Fold
- OS: iOS18, Android15

**Comment by aiya000** [comment]
Oh, wait please.
I tried a screenshot, using my iPhoneX and my Android15.
But it's succeed.
I'll debug a little more.

**Comment by NotOfficer** [comment]
The video isn't corruped, your android phone simply can't play it.

**Comment by aiya000** [comment]
What do you mean?
In another case (I'm in testing now, but), such as from iPhoneX to Pixel 9 Pro Fold, the movies completely can play, also the thumbnail can be generated successfully.

**Comment by Tienisto** [comment]
Is the behavior consistent, meaning are all videos coming from your iPhone15 become corrupted?
Can you check the videos from iPhoneX / iPhone11, are they .mov or .mp4? On iOS, you can configure to record mp4 instead of mov.

**Comment by aiya000** [comment]
I checked again with no mp4 config (with mov config).
As a result, I found out that all .mov are corrupted.

(FStop and Google Photos were smarter than expected and made them playable.
This time I checked with X-plorer.
From now on, I will check with X-plorer.
Sorry.)

- from iPhoneX (.mov): corrupted
- from iPhone11 (.mov): corrupted
- from iPhone15 (.mov): corrupted

But, I found that Google Photos can play that videos, and can generate that thumbnails.
FStop can only play, cannot generate that thumbnails.

I also checked a downloaded .mp4.
As a result, it is not corrupted.

- - -

To sum up:

- All .mov that filmed by my iPhone are corrupted (Checked by X-plorer)
    - But these can play on FStop and Google Photos
    - These thumbnails can be generated on Google Photos

- - -
 
Thanks!

![Screenshot_20241030-233239](https://github.com/user-attachments/assets/1dbb6f61-5da2-426d-a621-53da51d8e531)

**Comment by Tienisto** [comment]
Alright, so the mov files are not corrupted, they just can't be played by your X-plorer.

**Comment by aiya000** [comment]
So does that mean the real problem is that FStop can't output thumbnails?

**Comment by Tienisto** [comment]
It seems so, unless you have a mov that can be played by X-plorer.

**Comment by NotOfficer** [comment]
.mov is just the video container. your phone is not able to handle specific codec that your iphone produces (probably HEVC/hvc1/H.265).

**Comment by aiya000** [comment]
Thank you.
I comfirmed 'Most Compatible' .mov can play and can generate the thumbnails.

This problem I described is not LocalSend's problem.

Thanks a lot!
